app/controllers/api/services_controller.rb

```diff
@@ -8,7 +8,7 @@ class Api::ServicesController < Api::BaseController
   activate_menu :serviceadmin, :overview
 
   before_action :deny_on_premises_for_master
-  before_action :authorize_section
+  before_action :authorize_section, except: :index
```

Comment:
This is to fix the Access Denied when trying to access the list of products with any service-based permission other than `:plans`.